light-curve/light_curve/light_curve_py/features/rainbow/symmetric.py

```diff
@@ -0,0 +1,220 @@
+        `t` must be sorted in ascending order. If `check` is True, the input is checked for
+        NaNs and Infs. If `fill_value` is not None, it is used to fill the output array if
+        the feature cannot be evaluated.
+
+    model(t, band, *params)
+        Evaluate Rainbow model on the given arrays of times and bands. `*params` are
+        fit parameters, basically the output of `__call__` method but without the last
+        parameter (reduced Chi^2 of the fit). See parameter names in the `.name` attribute.
+
+    peak_time(*params)
+        Return bolometric peak time for given set of parameters
+    """
+
+    with_temperature_evolution: bool = dataclass_field(default=True, kw_only=True)
+    """Whether to include a temperature evolution in the fit."""
+    with_rise_only: bool = dataclass_field(default=False, kw_only=True)
+    """Whether to use sigmoid (rising part only) bolometric function in the fit."""
+
+    @staticmethod
+    def _common_parameter_names() -> List[str]:
+        return ["reference_time"]
+
+    def _bolometric_parameter_names(self) -> List[str]:
+        if self.with_rise_only:
+            return ["amplitude", "rise_time"]
+        else:
+            return ["amplitude", "rise_time", "fall_time"]
+
+    def _temperature_parameter_names(self) -> List[str]:
+        if self.with_temperature_evolution:
+            return ["Tmin", "Tmax", "k_sig"]
+        else:
+            return ["Tmin"]
+
+    def bol_func(self, t, params):
+        if self.with_rise_only:
+            t0, amplitude, rise_time = params[self.p.all_bol_idx]
+        else:
+            t0, amplitude, rise_time, fall_time = params[self.p.all_bol_idx]
+        dt = t - t0
+        result = np.zeros_like(dt)
+
+        # To avoid numerical overflows, let's only compute the exponents not too far from t0
+        if self.with_rise_only:
+            idx = dt > -100 * rise_time
+            result[idx] = amplitude / (np.exp(-dt[idx] / rise_time) + 1)
+        else:
+            idx = (dt > -100 * rise_time) & (dt < 100 * fall_time)
+            result[idx] = amplitude / (np.exp(-dt[idx] / rise_time) + np.exp(dt[idx] / fall_time))
+
+        return result
+
+    def temp_func(self, t, params):
+        if self.with_temperature_evolution:
+            t0, T_min, T_max, k_sig = params[self.p.all_temp_idx]
+            dt = t - t0
+
+            result = np.zeros_like(dt)
+
+            # To avoid numerical overflows, let's only compute the exponent not too far from t0
+            idx1 = dt <= -100 * k_sig
+            idx2 = (dt > -100 * k_sig) & (dt < 100 * k_sig)
+            idx3 = dt >= 100 * k_sig
+
+            result[idx1] = T_min
+            result[idx2] = T_min + (T_max - T_min) / (1.0 + np.exp(dt[idx2] / k_sig))
+            result[idx3] = T_max
+
+            return result
+        else:
+            t0, T_min = params[self.p.all_temp_idx]
+
+            return T_min
```

Comment:
We don't need this `else` here since we return from `if`

```suggestion

        t0, T_min = params[self.p.all_temp_idx]
        return T_min
```